Author: <csmall@debian.org>
Description: Fixes groff problems
Removes pslab as it doesnt appear
Index: b/vmstat.8
===================================================================
--- a/vmstat.8	2009-11-24 20:53:04.000000000 +1100
+++ b/vmstat.8	2009-11-24 21:00:36.000000000 +1100
@@ -1,6 +1,6 @@
 .\"  This page Copyright (C) 1994 Henry Ware <al172@yfn.ysu.edu>
 .\"  Distributed under the GPL, Copyleft 1994.
-.TH VMSTAT 8 "27 July 1994 " "Throatwobbler Ginkgo Labs" "Linux Administrator's Manual"
+.TH VMSTAT 8 "2009 Jan 9" "Throatwobbler Ginkgo Labs" "Linux Administrator's Manual"
 .SH NAME
 vmstat \- Report virtual memory statistics
 .SH SYNOPSIS
@@ -22,32 +22,35 @@
 .RB [ "\-d"]
 .br
 .B vmstat
+.RB [ "\-D"]
+.br
+.B vmstat
 .RB [ "\-p disk partition"]
 .br
 .B vmstat
 .RB [ "\-V" ]
 .SH DESCRIPTION
 \fBvmstat\fP reports information about processes, memory, paging,
-block IO, traps, and cpu activity.
+block IO, traps, disks and cpu activity.
 
 The first report produced gives averages since the last reboot.  Additional
 reports give information on a sampling period of length \fIdelay\fP.
 The process and memory reports are instantaneous in either case.
 
 .SS Options
-The \fB-a\fP switch displays active/inactive memory, given a 2.5.41 kernel or better.
+The \fB\-a\fP switch displays active/inactive memory, given a 2.5.41 kernel or better.
 .PP
-The \fB-f\fP switch displays the number of forks since boot.
+The \fB\-f\fP switch displays the number of forks since boot.
 This includes the fork, vfork, and clone system calls, and is
 equivalent to the total number of tasks created. Each process
 is represented by one or more tasks, depending on thread usage.
 This display does not repeat.
 .PP
-The \fB-m\fP displays slabinfo.
+The \fB\-m\fP displays slabinfo.
 .PP
-The \fB-n\fP switch causes the header to be displayed only once rather than periodically.
+The \fB\-n\fP switch causes the header to be displayed only once rather than periodically.
 .PP
-The \fB-s\fP switch displays a table of various event counters
+The \fB\-s\fP switch displays a table of various event counters
 and memory statistics. This display does not repeat.
 .PP
 .I delay
@@ -58,13 +61,15 @@
 is the number of updates.  If no count is specified and delay is
 defined, \fIcount\fP defaults to infinity.
 .PP
-The \fB-d\fP reports disk statistics (2.5.70 or above required) 
+The \fB\-d\fP reports disk statistics (2.5.70 or above required)
+.PP
+The \fB-D\fP reports some summary statistics about disk activity.
 .PP
-The \fB-p\fP followed by some partition name for detailed statistics (2.5.70 or above required) 
+The \fB\-p\fP followed by some partition name for detailed statistics (2.5.70 or above required)
 .PP
-The \fB-S\fP followed by k or K or m or M switches outputs between 1000, 1024, 1000000, or 1048576 bytes 
+The \fB\-S\fP followed by k or K or m or M switches outputs between 1000, 1024, 1000000, or 1048576 bytes
 .PP
-The \fB-V\fP switch results in displaying version information.
+The \fB\-V\fP switch results in displaying version information.
 .PP
 .SH FIELD DESCRIPTION FOR VM MODE
 .SS
@@ -81,8 +86,8 @@
 free: the amount of idle memory.
 buff: the amount of memory used as buffers.
 cache: the amount of memory used as cache.
-inact: the amount of inactive memory. (-a option)
-active: the amount of active memory. (-a option)
+inact: the amount of inactive memory. (\-a option)
+active: the amount of active memory. (\-a option)
 .fi
 .PP
 .SS
@@ -110,9 +115,9 @@
 .B "CPU "
 These are percentages of total CPU time.
 .nf
-us: Time spent running non-kernel code. (user time, including nice time)
+us: Time spent running non\-kernel code. (user time, including nice time)
 sy: Time spent running kernel code. (system time)
-id: Time spent idle. Prior to Linux 2.5.41, this includes IO-wait time.
+id: Time spent idle. Prior to Linux 2.5.41, this includes IO\-wait time.
 wa: Time spent waiting for IO. Prior to Linux 2.5.41, included in idle.
 st: Time stolen from a virtual machine. Prior to Linux 2.6.11, unknown.
 
@@ -161,8 +166,6 @@
 total: Total number of available objects
 size: Size of each object
 pages: Number of pages with at least one active object
-totpages: Total number of allocated pages
-pslab: Number of pages per slab
 .fi
 
 .SH NOTES
